documentation/ame-guide/modules/ROOT/pages/modeling/start-new-models.adoc

```diff
@@ -5,24 +5,24 @@ To start modeling, proceed as follows:
 
 image:create-new-model.png[width=30%]
 
-. Click on the menu on *FILE* -> *New...* xref:getting-started/ui-overview.adoc#menu-file[File menu].
+. Click the xref:getting-started/ui-overview.adoc#menu-file[File menu] and select *New...*.
```

Comment:
see comments above to this section

Reply:
Wait is this a duplicated section?

yes -- that same content sits in "Load and save models" and in "start-new-models".

Follow-up issue please to reshape that. 

![grafik](https://github.com/eclipse-esmf/esmf-aspect-model-editor/assets/101344507/662b431d-6754-402e-8e91-b2d94b6b5e43)